src/gc.c

```diff
@@ -7,6 +7,7 @@
 #ifdef __GLIBC__
 #include <malloc.h> // for malloc_trim
 #endif
+#include <stdbool.h>
```

Comment:
Can we just use an `int`? All of the C codebase uses `int` instead of `bool`, so it's desirable to keep the consistency here.